components/tikv_util/src/yatp_pool/future_pool.rs

```diff
@@ -210,9 +239,10 @@ impl PoolInner {
         F::Output: Send,
     {
         let metrics_handled_task_count = self.env.metrics_handled_task_count.clone();
-        let metrics_running_task_count = self.env.metrics_running_task_count.clone();
+        let metrics_running_task_count =
+            self.env.metrics_running_task_count_by_priority[TaskPriority::Medium as usize].clone();
```

Comment:
Why is `Medium` always in here?

Reply:
Because these futures are not spawned with the Extras parameter, so just fallback to default medium priority. `spawn_handle` is not used in the normal code path, so this kind of requests should not be a lot.